tbx/static_src/sass/components/navigation/_breadcrumbs-nav.scss

```diff
@@ -1,46 +1,32 @@
 @use "config" as *;
 
 .breadcrumbs-nav {
+    @include font-size(supporting);
     display: flex;
-    align-items: center;
     flex-direction: row;
+    flex-wrap: wrap;
     width: 100%;
-    height: 100%;
-    margin-bottom: 10px;
+    margin: 15px 0 5px; // bottom margin of 5px is a 15px margin minus the 10px bottom margin on the li.
```

Comment:
Question: could this use `$spacer-mini` for 15px or is the spacing not part of the vertical spacing units?

Reply:
good point - I've updated that. I've also removed some yaml variables that weren't being used.